superset-frontend/src/components/ReportModal/HeaderReportDropdown/index.tsx

```diff
@@ -204,7 +223,14 @@ export default function HeaderReportDropDown({
     ) : (
       <Menu selectable={false} css={onMenuHover}>
         <Menu.Item onClick={handleShowMenu}>
-          {t('Set up an email report')}
+          {DropdownItemExtension ? (
+            <StyledDropdownItemWithIcon>
+              <div>{t('Set up an email report')}</div>
+              <DropdownItemExtension />
```

Comment:
Do we want to be this specific, or could we use an extension for the whole item, passing down the text instead?

Reply:
I'm not sure here.  On one hand having the extension be a whole replacement would allow for much more flexibility for anyone looking to use it.  On the other hand, having it be specific provides one rigid supported pattern for the extension, which could be seen as a benefit.